thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -44,7 +44,9 @@
   <h3>Dimensions</h3>
   {{rootcause-heatmap
     breakdowns=breakdowns
-    selectedUrns=selectedUrns
+    currentUrns=heatmapCurrentUrns
+    current2baseline=heatmapCurrent2Baseline
```

Comment:
"currentUrns" and "current2baseline" are gone

Reply:
oh oops, didn't push the revised change. thanks for catching